android/build.gradle

```diff
@@ -26,5 +26,5 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    api 'io.sentry:sentry-android:5.0.0-beta.4'
+    api 'io.sentry:sentry-android:5.0.0'
```

Comment:
it'd be nice to address the breaking changes too, see https://github.com/getsentry/sentry-dart/pull/486/files
eg renaming `cacheDirSize` to `maxCacheItems`

Reply:
RN didn't support the `cacheDirSize` option yet, which I guess is a downside of having to set each option individually when we have to maintain multiple hybrid SDKs... On a related note but off topic from this PR, @bruno-garcia suggested we should take a map for options from hybrid Sdks

Reply:
yep, indeed, this is blocked by https://github.com/getsentry/sentry-java/issues/1117 though
so all good for now